### DISCUSSION TITLE
Applet GJS documentation deviation

### DISCUSSION THREAD

**Discussion by ghost** [linuxmint/discussions#715] [Development]
I have installed devhelp, but when I browse it there are missing APIs for CJS; for example, there is no `ui.popupMenu.PopupMenuManager` there, even though my Linux Mint version (Linux Mint 22 Cinnamon)'s `menu@cinnamon.org` applet uses it.

Also, there is no consistent documentation for St (GNOME Shell Tookit) API in CJS for example. There is `IconStyle` [here](https://gjs-docs.gnome.org/st15~15/), but not `IconType`, which the applet above uses.

**Comment by mtwebster** [comment]
Cjs is not an API, it is javascript language bindings. The apis you're referring to mostly live in `https://github.com/linuxmint/cinnamon/tree/master/js` (js/ui/popupMenu.js for example).

You can get quite a bit by installing the cinnamon-doc package (if you're in Mint).

Some API that lives in muffin (Cinnamon's window manager) is best viewed at https://gjs-docs.gnome.org/ using API 7 libraries. Importable config: [devdocs.json](https://github.com/user-attachments/files/18199065/devdocs.json). Unfortunately these aren't available in devhelp.

**Reply by ghost** [reply]
I've tried installing the cinnamon-doc package, but the `/usr/share/doc/cinnamon-doc` directory contains... Nothing, just a readme and a license! Where would I access that exactly?
